src/Infiltration/formulas/game.ts

```diff
@@ -2,35 +2,40 @@ import { Player } from "@player";
 import { clampNumber } from "../../utils/helpers/clampNumber";
 
 export const MaxDifficultyForInfiltration = 3.5;
-const MaxEffectTime = 30 * 60 * 1000; // 30 minutes in milliseconds
-const date = globalThis.Date;
```

Comment:
I reverted this part in my change because I don't think we should be going out of our way for this sort of thing. There are a myriad of ways that players can attack the game through prototypes, we can't defend against all of them, and this one is relatively benign compared to just speeding the engine ticks up.